### PR TITLE
log: Add config option

### PR DESCRIPTION
**Pull request by n-riesco**
* Added debug option, so that it's possible to enable debug messages on
  a running session.

### CODE REVIEW

**Comment by lgeiger** [comment]
Excellent idea! That's a lot easier 🎉 

Would you like to update the [debugging guide](https://github.com/nteract/hydrogen/blob/master/TROUBLESHOOTING.md#debugging-guide) as well?

**Comment by n-riesco** [comment]
@lgeiger Shall I remove all the info about the opening atom in dev mode?

**Comment by lgeiger** [comment]
> @lgeiger Shall I remove all the info about the opening atom in dev mode?

Yeah, I would replace it with the new config option.

File: lib/config.js [hunk 1]
```diff
@@ -16,35 +16,49 @@ const Config = {
   schema: {
     autocomplete: {
       title: 'Enable Autocomplete',
+      includeTitle: false,
```
Comment (lgeiger) [inline lib/config.js]: What does `includeTitle: false` do? I couldn't find anything in the Atom docs

Comment (n-riesco) [inline lib/config.js]: I had to [read the code](https://github.com/atom/settings-view/blob/197f962fec680e1b9b1c7c8f5fd162a99731d959/lib/settings-panel.coffee#L48-L69) to fix this issue.

Comment (n-riesco) [inline lib/config.js]: I'll try to get a PR merged in Atom after I'm done publishing an update for IJavascript.

Comment (lgeiger) [inline lib/config.js]: Interesting, on my machine I don't see a difference between `includeTitle: false,` and the default setting.

If it fixes a issue on your machine, let's go with it 👍

**Pull request merged**